flink-kubernetes/src/main/java/org/apache/flink/kubernetes/configuration/KubernetesConfigOptions.java

```diff
@@ -220,7 +221,9 @@
                     .withDescription(
                             Description.builder()
                                     .text(
-                                            "The cluster-id, which should be no more than 45 characters, is used for identifying a unique Flink cluster. "
+                                            "The cluster-id, which should be no more than "
+                                                    + KubernetesLabel.getClusterIdMaxLength()
```

Comment:
We could us the `%s` placeholder here like it's done for the other variables to be more consistent.